localstack/services/firehose/firehose_api.py

```diff
@@ -39,6 +43,33 @@ def put_record(stream_name, record):
 def put_records(stream_name, records):
     stream = get_stream(stream_name)
     for dest in stream['Destinations']:
+        if 'ESDestinationDescription' in dest:
+            es_dest = dest['ESDestinationDescription']
+            es_index = es_dest['IndexName']
+            es_type = es_dest['TypeName']
+            es = connect_elasticsearch()
+            for record in records:
+                data = base64.b64decode(record['Data'])
+                unparsed_body = json.loads(data)
+                keys = unparsed_body.get('Keys')
```

Comment:
It seems that this code is specifically targeting records coming from a DynamoDB stream, is that right? I guess we should try and keep the code in here as generic as possible, to be able to handle arbitrary records being written to the Firehose stream.